libraries/AP_HAL_SITL/Scheduler.cpp

```diff
@@ -32,15 +32,15 @@ void Scheduler::init()
 {
 }
 
-void Scheduler::delay_microseconds(uint16_t usec)
+void Scheduler::delay_microseconds(uint16_t us)
```

Comment:
I rather prefer the former as it is in line with other places from the code and Linux/unix structs such as timespec.